CHANGELOG.md

```diff
@@ -6,21 +6,21 @@
 - [](https://github.com/vegaprotocol/vega/issues/xxxx) -
 
 ### 🗑️ Deprecation
-- [](https://github.com/vegaprotocol/vega/issues/xxxx) -
+- [7068](https://github.com/vegaprotocol/vega/issues/7068) - Alias `vegawallet info` to `vegawallet describe`, before definitive renaming.
```

Comment:
?

Reply:
yea I moved them around because they were put on the wrong release, noticed it while adding the entries for this.